platform/linux-generic/pktio/loop.c

```diff
@@ -162,7 +162,6 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 						       pkt_len, seg_len,
 						       &new_pool, pkt_hdr, true);
 			if (ret) {
-				failed++;
```

Comment:
It would good to separation parse and cls as part of this PR. classify_packet() could fail for other reasons than "drop CoS", but in practice it does not do that today as it does not check return value of parse. Also, packet_parse_reset() is now called twice on this CLS path.

It would be cleaner do parse call outside of classify_packet(). That way it would be clean to increment this counter when there are parse errors, but not when a "drop CoS" is hit.

Reply:
These are addressed in #1499 and #1505.